druid-shell/src/window.rs

```diff
@@ -65,10 +65,18 @@ impl IdleHandle {
     where
         F: FnOnce(&dyn Any) + Send + 'static,
     {
-        self.0.add_idle(callback)
+        self.0.add_idle_callback(callback)
+    }
+
+    pub fn schedule_idle(&mut self, token: IdleToken) {
```

Comment:
This should have some basic doc string, maybe:

"Request a callback from the runloop. Your [`WinHander::idle`](trait.WinHandler.html#tymethod.idle) method will be called with the `token` that was passed in."